hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryRunner.java

```diff
@@ -144,6 +145,31 @@ public Result runPartitionIndexOrPartitionScanQueryOnGivenOwnedPartition(Query q
         return populateResult(query, partitions, entries);
     }
 
+    // MIGRATION UNSAFE QUERYING - MIGRATION STAMPTS ARE NOT VALIDATED, so assumes a run on partition-thread
+    // for a single partition. If the index is global it won't be asked
+    public Result runPartitionIndexOrPartitionScanQueryOnGivenOwnedPartition(Query query, Collection<Integer> partitions) {
```

Comment:
can be a shorter method name if possible

Reply:
Uff, good I deleted the method earlier :)